spec.emu

```diff
@@ -234,6 +234,81 @@ contributors: Google, Ecma International
     <emu-note>
       The content in this table is proposed by Era Code WG to CLDR TC in <a href="https://github.com/unicode-org/cldr/pull/2665">PR 2665</a> and under review. It is targeted to be part of CLDR 43, scheduled to be released in April 2023.
     </emu-note>
+    <emu-table id="table-epoch-years">
+      <emu-caption>epoch years</emu-caption>
+      <table class="real-table">
+        <thead>
+          <tr>
+            <th>Calendar</th>
+            <th>Epoch ISO Year</th>
+          </tr>
+        </thead>
+        <tr>
+          <td>*"buddhist"*</td>
+          <td>-543</td>
+        </tr>
+        <tr>
+          <td>*"chinese"*</td>
+          <td>0</td>
+        </tr>
+        <tr>
+          <td>*"coptic"*</td>
+          <td>284</td>
+        </tr>
+        <tr>
+          <td>*"dangi"*</td>
+          <td>0</td>
+        </tr>
+        <tr>
+          <td>*"ethiopic"*</td>
+          <td>8</td>
```

Comment:
Likewise, should be 7?
```js
> new Date('0007-08-29').toLocaleString('en-ca-u-ca-ethiopic', { dateStyle: 'short' })
'1/1/5500 ERA0'
```
5500 is the last year of AA era, so arithmetic year 0. 1 AM (arithmetic year 1) starts in ISO year 8.